### PR TITLE
Improved Accuracy of the Model

### LINKED ISSUE #3: Improve Accuracy of the Model
**Describe the enhancement**
Improve the accuracy of the model. Change the **epochs** to 100 and the **learning rate** to 0.00001. If that is not enough, change the **epochs** to 190 and the **learning rate** to 0.00000001.

### PR DESCRIPTION
**Pull request by sagedemage**
Improved the accuracy of the model.

**Extra Tasks Completed**:
1. modified the model so it works with `CrossEntropyLoss`
2. increased the epochs to `190` and decreased the learning rate to `0.00000001`
3. the main program converts the model output to a NumPy `ndarray` in order to get a list of 1 predicted label
4. changed the batch size and model output to `37` to represent there are 37 labels of **pet breeds**
5. added links to **two images** for each breed in the CSV file to use it for the main program to display two images of the predicted breed of the pet
6. require the user to pick and save **one pet image** for the gui program
7. added key bindings for the GUI program

### CODE REVIEW

**Review by sagedemage — COMMENTED** [review]
LGTM!
